src/main/java/com/google/sps/servlets/UserCardsServlet.java

```diff
@@ -72,16 +75,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     if (userService.isUserLoggedIn()) {
 
-        String textNotTranslated = request.getParameter("textNotTranslated");
-
         // if blobKey is null, user did not add an image
         String folderKey = request.getParameter("folderKey");
         String blobKey = getBlobKeyfromBlobstore(request, "image");
         String labels = request.getParameter("labels");
         String fromLang = request.getParameter("fromLang");
         String toLang = request.getParameter("toLang");
         String textNotTranslated = request.getParameter("textNotTranslated");
```

Comment:
`rawText`? For both the query parameter and the variable name.

Reply:
Fixed in #10